app/client/src/components/editorComponents/CodeEditor/EvaluatedValuePopup.tsx

```
@@ -66,7 +66,6 @@ const CodeWrapper = styled.pre<{ colorTheme: EditorTheme }>`
   margin: 0px 0px;
   background-color: ${props => THEMES[props.colorTheme].editorBackground};
   color: ${props => THEMES[props.colorTheme].editorColor};
-  overflow: scroll;
```

We should hide the scrollbars for firefox browsers as well like we do for webkit
```
scrollbar-width: none;
```

---

Just curious: Why are we hiding scrollbars altogather? When the content is large, I noticed on chrome that we can scroll fine but a scrollbar is not shown (giving no sense of how much content is there and to what point I've scrolled to).

---

Comes down to asthetics I guess. The default scrollbars don't look nice with our designs so we are hiding them as of now. Another solution is to style the scroll like we do for the property pane today

---

Fair enough. I'll wait for those shiny new scrollbars then :)

I've made the change you suggested.